website/docs/intro/whats-new.mdx

```diff
@@ -0,0 +1,178 @@
+---
+sidebar_position: 2
+sidebar_label: What's new in version 1.7?
+description: |-
+  Learn all about the new features in OpenTofu 1.7.
+---
+
+# What's new in OpenTofu 1.7?
+
+This page will run you through the most important changes in OpenTofu 1.7:
+
+- [Loopable import blocks](#loopable-import-blocks)
+- [State encryption](#state-encryption)
+- [Removed block](#removed-block)
```

Comment:
This is missing functions.

Reply:
Also, I'd reorder:
- State Encryption
- Functions
- Removed Block
- Loopable import blocks

Reply:
That is, provider-defined functions